crates/humanode-peer/src/chain_spec.rs

```diff
@@ -275,6 +283,10 @@ fn testnet_genesis(
                         humanode_runtime::TokenClaimsPot::account_id(),
                         INITIAL_POT_ACCOUNT_BALANCE,
                     ),
+                    (
+                        humanode_runtime::NativeToEvmSwapBridgePot::account_id(),
+                        INITIAL_POT_ACCOUNT_BALANCE,
```

Comment:
This should be an existential balance + sum of all endowed EVM accounts excluding the bridge pot.

Reply:
https://github.com/humanode-network/humanode/pull/673/commits/0dda71e2cb28c04743c355dae590571cbfead814

Reply:
https://github.com/humanode-network/humanode/pull/673/commits/0c72e91188f014f347f75923cd07d1b37f2a9043

Reply:
Will be resolved as part of #695.